serverlessworkflow/modules/ROOT/pages/supporting-services/jobs-service.adoc

```diff
@@ -0,0 +1,101 @@
+= Job Service
+:compat-mode!:
+// Metadata:
+:description: Job Service to control timeouts in {context}
+:keywords: kogito, workflow, serverless, timeout, timer, expiration, job service
+// links
+:jobs_service_image_url: https://quay.io/repository/kiegroup/kogito-jobs-service-allinone
+:jobs_service_image_usage_url: https://github.com/kiegroup/kogito-images#jobs-services-all-in-one
+
+In {product_name} architecture there is a dedicated supporting service that is responsible for controlling the execution of
+time-triggered actions, also known as jobs.
+
+The job execution is a callback to the {context} runtime application. This callback can be configured in different ways as described in the <<job-service-communication, communication>> section.
+
+[#job-service]
+== Job Service configuration
+
+All timer nodes that might be declared in a workflow, are handled by Job Service, which is responsible for managing, scheduling, and firing all actions (jobs) to be executed in the workflows.
+
+Suppose the workflow service is not configured to use job service or there is no such service running. In that case, all timer-related actions use an embedded in-memory implementation of job service, which should not be used in production, since when the application shutdown, all timers are lost, which in a serverless architecture is a very common behavior with the scale to zero approach. That said, the no job service configuration can only be used for testing or development, but not for production.
+
+The main goal of the Job Service is to work with only active jobs. The Job Service tracks only the jobs that are scheduled and that need to be executed. When a job reaches a final state, the job is removed from the Job Service.
+
+When configured in your environment, all the jobs information and status changes are sent to the {product_name} `Data
+Index Service`, where they can be indexed and made available by GraphQL queries.
+
+[NOTE]
+====
+Data index service and the support for jobs information will be available in future releases.
+====
+
+=== Job Service persistence
+
+An important configuration aspect of job service is the persistence mechanism, where all job information is stored in a database that makes this information durable upon service restarts and guarantees no information is lost.
+
+[#job-service-postgresql]
+==== PostgreSQL
+
+PostgreSQL is the recommended database to use with job service.
+Additionally, it provides an initialization procedure that integrates link:https://flywaydb.org[Flyway] for the database initialization. It automatically controls the database schema, in this way all tables are created by the service.
+
+In case you need to externally control the database schema, you can check the Flyway SQL scripts in link:{kogito_apps_url}/jobs-service/jobs-service-postgresql/src/main/resources/db/migration[migration] and apply them.
```

Comment:
I believe this directory has now been renamed to: jobs-service so it might be needed to update this link too.

Reply:
fixed, thanks.